pom.xml

```
@@ -108,6 +114,8 @@
                 <extensions>true</extensions> <!-- Support Bundle Packaging -->
                 <configuration>
                     <instructions>
+                        <!-- Relax dependency version in order to allow run inside multiple versions of nexus -->
+                        <_consumer-policy>$(version;==)</_consumer-policy>
```

i think it should be like this:
```suggestion
                        <_consumer-policy>$${version;==}</_consumer-policy>
```

---

Done